### ISSUE TITLE
Handle MPANs with missing leading zeros

### ISSUE THREAD

**Issue by joshdunnlime** [limejump/mpan#8]
MPAN function accepts int and cast them to str, however, some MPANs contain leading zeros, as such, these MPANs break the checksum and length checks.

e.g. `str(int("00123412341234"))` becomes `"123412341234"`. It is a trivial fix to prepend the zeros and run the checksum. If it passes, we can be pretty sure this is the correct fix for the MPAN.

**Comment by limedaniel** [comment]
I feel terrible.  This is the first issue created for this library from someone else at the company and I have to reject it.  I'm so sorry.

The problem I have with this is that MPANs are not integers.  While they're often composed entirely of numbers, this (a) isn't always the case (sometimes there's letters!), and (b) the fact that they're mostly numbers doesn't make the MPAN an integer: it's simply a series of numeric fields glued together with no separators.  For this reason, treating it like an integer isn't something we should be encouraging.

I also have some deep objections to the idea that a library designed to *parse* a string would *alter* that string as part of the parsing process.  That's a big no-no in my book.

So in summary, my advice is that if you're currently casting an MPAN as an int in your own code, you should definitely stop doing that 'cause LLFCs are going to cause you pain one day, and if you've got an int that you want to treat like an MPAN and are comfortable with the idea of just adding zeroes to the left, then just use some string formatting on it before you pass it in:

```python
my_int_mpan_as_a_string = str(my_int_mpan).rjust(21, "0")
MPAN(my_int_mpan_as_a_string)
```
